packages/completer/src/handler.ts

```diff
@@ -477,6 +486,85 @@ export namespace CompletionHandler {
     connector: IDataConnector<IReply, void, IRequest>;
   }
 
+  /**
+   * Wrapper object for ICompletionItem.
+   * Implementers of this interface should be responsible for
+   * deduping and sorting the items in the list.
+   */
+  export interface ICompletionItems {
```

Comment:
> My only concern, with the documentation use case in mind, is that if issues arise requiring additional enhancements, they would require a breaking change, potentially delaying fixes. Happy to remove this if it's an over optimization.

Yeah that's fair... I would say that at the moment the simplicity of removing it would be nice, unless we can imagine how we would extend it now.